languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/GermanSpellerRule.java

```diff
@@ -2923,6 +2924,9 @@ protected List<SuggestedReplacement> getOnlySuggestions(String word) {
     if (RECHTMASIG_WITH_CASES.matcher(word).matches()) {
       return topMatch(word.replaceFirst("mässig", "mäßig"));
     }
+    if (CONTAINS_MASZNAME.matcher(word).matches()) {
+      return topMatch(word.replaceFirst("aßname", "aßnahme"));
```

Comment:
I think you can use `a(ss|ß)name` here as the first argument. And `testGetAdditionalTopSuggestions()` in the test should be extended with 2-3 examples maybe.